### ISSUE TITLE
incorrect peer dependency

### ISSUE THREAD

**Issue by pangeco** [naveenvignesh5/react-native-otp-textinput#52]
warning : react-native-otp-textinput@1.1.5" has incorrect peer dependency "react-native@^0.72.4".
this happen when using newer version of react-native

react-native : "0.73.3"

**Comment by sampadanaik** [comment]
+1

**Comment by Reotech** [comment]
+1

**Comment by Felipe-OT** [comment]
+1

**Comment by dennisnjd** [comment]
+1 . Please do something.

**Comment by IsmailJanParacha** [comment]
Did any fix this issue? Temporary we can ignore this error by using the force command

**Comment by Reotech** [comment]
Yeah I fixed mine by editting the version to match the required version in package-lock.json

**Comment by IsmailJanParacha** [comment]
@Reotech, It would be appreciated if you share the compatible version

**Comment by Reotech** [comment]
Okay

"node_modules/react-native-otp-textinput": {
      "version": "1.1.5",
      "resolved": "https://registry.npmjs.org/react-native-otp-textinput/-/react-native-otp-textinput-1.1.5.tgz",
      "dependencies": {
        "react": "^18.2.0"
      },
      "peerDependencies": {
        "react-native": "^0.73.4"
      }
    } so this is an example of mine, note the react-native version

**Comment by IsmailJanParacha** [comment]
Thank you @Reotech  it worked. Basically, the react native version was not matching in peer dependency so I matched it and then it worked

**Comment by Reotech** [comment]
Great!
You're welcome @IsmailJanParacha
